server/src/test/java/org/opentosca/toscana/plugins/cloudformation/CFBuilderTest.java

```diff
@@ -0,0 +1,88 @@
+package org.opentosca.toscana.plugins.cloudformation;
+
+import com.scaleset.cfbuilder.core.Module;
+import com.scaleset.cfbuilder.core.Template;
+import com.scaleset.cfbuilder.ec2.Instance;
+import com.scaleset.cfbuilder.ec2.SecurityGroup;
+import com.scaleset.cfbuilder.ec2.SecurityGroupIngress;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+
+public class CFBuilderTest extends Module {
```

Comment:
Please rename this class!

If you want to use abbreviations please consider using CFN instead of CF to prevent confusion with CloudFoundry!